bin/rails

```diff
@@ -1,8 +1,4 @@
 #!/usr/bin/env ruby
-begin
```

Comment:
Yeah i did remove it, i couldn't get it playing nicely with cellect and it's connection to zk on app boot when using docker-compose for testing. See [this commit](https://github.com/camallen/Panoptes/commit/47f9d82009fff8bf1e3ad365a930d304b92dcb2b), happy to put it back in but the Cellect NodeSet (client in this case) but had dead actors due the app creating a ZK connection on boot.